build/kiali/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG VERSION
+
+FROM quay.io/kiali/kiali:${VERSION} as console
+
+FROM golang:1.14 as builder
+
+ARG VERSION
+
+RUN set -eux; \
+    \
+    apt-get update; \
+    apt-get install git;
```

Comment:
I think git is already installed in this image by default so we can probably remove this if that is confirmed.